docs/designers-developers/developers/data/data-core-block-editor.md

```diff
@@ -158,6 +158,32 @@ Returns the number of blocks currently present in the post.
 
 Number of blocks in the post.
 
+### getSelectionStart
+
+Returns the current selection start block client ID, rich text area ID and
+text index.
+
+*Parameters*
+
+ * state: Block editor state.
+
+*Returns*
+
+Selection start information.
```

Comment:
How about we add more details about the shape of the object returned here? I see it looks this way at the moment:

```js
start: {
	block: action.block,
	identifier: action.identifier,
	offset: action.start,
},
end: {
	block: action.block,
	identifier: action.identifier,
	offset: action.end,
},
```

This might be important for those who can work with those selectors to know the shape.